service/src/main/java/bio/terra/pipelines/stairway/imputation/steps/gcp/SubmitCromwellSubmissionStep.java

```diff
@@ -59,10 +71,65 @@ public StepResult doStep(FlightContext flightContext) {
     String description = inputParameters.get(JobMapKeys.DESCRIPTION.getKeyName(), String.class);
     String wdlMethodName =
         inputParameters.get(RunImputationJobFlightMapKeys.WDL_METHOD_NAME, String.class);
+    String wdlMethodVersion =
+        inputParameters.get(RunImputationJobFlightMapKeys.WDL_METHOD_VERSION, String.class);
+
+    List<PipelineInputDefinition> inputDefinitions =
+        inputParameters.get(
+            RunImputationJobFlightMapKeys.PIPELINE_INPUT_DEFINITIONS, new TypeReference<>() {});
+    List<PipelineOutputDefinition> outputDefinitions =
+        inputParameters.get(
+            RunImputationJobFlightMapKeys.PIPELINE_OUTPUT_DEFINITIONS, new TypeReference<>() {});
 
     // validate and extract parameters from working map
     FlightMap workingMap = flightContext.getWorkingMap();
 
+    // grab current method config and validate it
+    MethodConfiguration methodConfiguration =
+        rawlsService.getCurrentMethodConfigForMethod(
```

Comment:
do we want to wrap this in try-catch retry logic like the later rawls call?

Reply:
oh good point, definitely missed this.